airbyte-integrations/connectors/source-facebook-marketing/unit_tests/helpers.py

```diff
@@ -2,4 +2,14 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
-#
+import pytest
```

Comment:
I'm not sure why this file is here - I think we can remove it since the same thing is being tested in `test_utils`

Reply:
yes, forgot to delete😅